README.id.md

```diff
@@ -0,0 +1,203 @@
+# Contoh Spring Cloud 
+
+[![Status Build](https://travis-ci.com/naXa777/spring-cloud-example.svg?branch=master&style=flat)](https://travis-ci.com/naXa777/spring-cloud-example)
+
+Playground ini dipergunakan untuk pembelajaran [Spring Cloud](https://cloud.spring.io/) dan [Kotlin](https://kotlinlang.org/) berlandaskan contoh-contoh.
+
+_Baca ini dalam bahasa lain: [English](README.en.md) [На русском](README.md)_
+
+## Kontribusi
+
+&#x1F49A; Mulai [disini](https://github.com/naXa777/spring-cloud-example/issues).
+
+## Deskripsi
+
+> [Cloud-native](https://pivotal.io/de/cloud-native) merupakan pendekatan yang digunakan untuk membangun dan menjalankan aplikasi yang memanfaatkan keuntungan dari model komputasi awan.
+
+> [Microservices](https://pivotal.io/microservices) merupakan pendekatan yang digunakan untuk mengembangkan sebuah aplikasi sebagai suatu koleksi dari servis-servis kecil; setiap servis ini mengimplementasikan kapabilitas bisnis, jalan dalam proses tersendiri dan berkomunikasi melalui API HTTP. Setiap microservice dapat dideploy, upgrade, scale, dan restart namun tidak bergantung pada servis lain pada aplikasi tersebut, yang biasanya merupakan bagian dari sistem otomata.
+
+[(c) Pivotal](https://pivotal.io/de/cloud-native)
+
+Kita membagi aplikasi besar menjadi bagian-bagian yang lebih kecil yang disebut 'services'. Dan setiap dari servis ini dapat dideploy atau scale secara tersendiri._Bagaimana suatu servis mengetahui yang lainnya?_
```

Comment:
a whitespace is missing before `_Bagaimana`

Reply:
added whitespace, thanks for this. the symbol `_` is barely visible in my laptop.